torch/_dynamo/variables/dicts.py

```diff
@@ -194,14 +170,11 @@ def reconstruct(self, codegen):
             return [create_instruction("BUILD_MAP", arg=len(self.items))]
 
     @staticmethod
-    def _wrap_keys_python_var(d):
+    def _wrap_const_keys(d):
         """Wrap the keys of a dictionary with python objs as keys into Hashable objects"""
-        assert all(is_hashable_python_var(k) for k in d.keys())
+        assert all(ConstantVariable.is_literal(k) for k in d.keys())
```

Comment:
Curious - How does this still work? Earlier `is_hashable_python_var` supported way more than the set supported by `ConstantVariable.is_literal`.

Reply:
Note that we just call this function to wrap some strings in a `kwargs`. Let me remove this function completely and just do the transformation at the call site.